components/camel-spring-rabbitmq/src/main/java/org/apache/camel/component/springrabbit/SpringRabbitMQConstants.java

```diff
@@ -23,12 +23,47 @@ public final class SpringRabbitMQConstants {
     public static final String DEFAULT_EXCHANGE_NAME = "default";
 
     public static final String CHANNEL = "CamelSpringRabbitmqChannel";
-    @Metadata(description = "The exchange key.", javaType = "String")
+    @Metadata(description = "Producer: To override the endpoint configuration's routing key.", javaType = "String")
     public static final String ROUTING_OVERRIDE_KEY = "CamelSpringRabbitmqRoutingOverrideKey";
-    @Metadata(description = "The exchange name.", javaType = "String")
+    @Metadata(description = "Producer: To override the endpoint configuration's exchange name.", javaType = "String")
     public static final String EXCHANGE_OVERRIDE_NAME = "CamelSpringRabbitmqExchangeOverrideName";
-    @Metadata(description = "Delivery tag for manual acknowledge mode", javaType = "long")
+    @Metadata(description = "Consumer: Whether the message was previously delivered and requeued.", javaType = "Boolean")
```

Comment:
If it is only meant for the consumer, you could rather set the label to "consumer"

Reply:
done